awx/ui_next/src/components/JobList/JobListCancelButton.jsx

```diff
@@ -33,20 +40,40 @@ function JobListCancelButton({ i18n, jobsToCancel, onCancel }) {
   }, [isKebabified, isModalOpen, onKebabModalChange]);
 
   const renderTooltip = () => {
-    const jobsUnableToCancel = jobsToCancel
-      .filter(cannotCancel)
+    const cannotCancelPermissions = jobsToCancel
+      .filter(cannotCancelBecausePermissions)
+      .map(job => job.name);
+    const cannotCancelNotRunning = jobsToCancel
+      .filter(cannotCancelBecauseNotRunning)
       .map(job => job.name);
-    const numJobsUnableToCancel = jobsUnableToCancel.length;
+    const numJobsUnableToCancel = cannotCancelPermissions.concat(
+      cannotCancelNotRunning
+    ).length;
     if (numJobsUnableToCancel > 0) {
       return (
         <div>
-          {i18n._(
-            '{numJobsUnableToCancel, plural, one {You do not have permission to cancel the following job:} other {You do not have permission to cancel the following jobs:}}',
-            {
-              numJobsUnableToCancel,
-            }
+          {cannotCancelPermissions.length > 0 && (
+            <div>
+              {i18n._(
+                '{numJobsUnableToCancel, plural, one {You do not have permission to cancel the following job:} other {You do not have permission to cancel the following jobs:}}',
+                {
+                  numJobsUnableToCancel: cannotCancelPermissions.length,
+                }
+              )}
+              {' '.concat(cannotCancelPermissions.join(', '))}
+            </div>
+          )}
+          {cannotCancelNotRunning.length > 0 && (
+            <div>
+              {i18n._(
+                '{numJobsUnableToCancel, plural, one {You cannot cancel the following job because it is not running:} other {You cannot cancel the following jobs because they are not running:}}',
+                {
+                  numJobsUnableToCancel: cannotCancelNotRunning.length,
```

Comment:
@mabashian 
lgtm

Since we're just listing out the names, it seems the tooltip behavior would still be reasonable even when there's a _lot_ of jobs to list.